becquerel/core/peakfinder.py

```diff
@@ -14,6 +14,12 @@ class PeakFilterError(Exception):
     pass
 
 
+class PeakFindingWarning(UserWarning):
```

Comment:
Could you rename this `PeakFinderWarning` to be consistent with `PeakFinderError`? Might also make sense to move it down near `PeakFinderError` around line 155.

Reply:
@arunpersaud any chance you could make this change? Thanks!